packages/manager/modules/pci/src/projects/project/training/onboarding/onboarding.constants.js

```diff
@@ -6,8 +6,8 @@ export const GUIDES = [
       'public-cloud::pci::projects::project::training::onboarding::go-to-docs-tasks',
   },
   {
-    id: 'install-eai-cli',
-    link: 'https://docs.ovh.com/gb/en/ai-training/install-eai-client/',
+    id: 'install-cli',
```

Comment:
The `id` is used to generated translation keys.

see:
https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/training/onboarding/onboarding.controller.js#L22-L25

It means that few keys must be updated as well:

```sh
$ git grep "install-eai-cli" -- "*.json"
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_de_DE.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_title": "AI Training Client installieren",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_de_DE.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_description": "Erfahren Sie, wie Sie den AI Training Client schnell in Ihrer Arbeitsumgebung installieren können.",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_en_GB.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_title": "Install AI Training client",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_en_GB.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_description": "Learn how to quickly install the AI Training client on your work desktop.",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_es_ES.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_title": "Instalar el cliente AI Training",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_es_ES.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_description": "Descubra cómo instalar rápidamente el cliente AI Training en su entorno de trabajo.",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_fr_CA.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_title": "Installer le client AI Training",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_fr_CA.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_description": "Apprenez comment installer rapidement le client AI Training sur votre environnement de travail.",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_fr_FR.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_title": "Installer le client AI Training",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_fr_FR.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_description": "Apprenez comment installer rapidement le client AI Training sur votre environnement de travail.",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_it_IT.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_title": "Installa il client AI Training",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_it_IT.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_description": "Scopri come installare rapidamente il client AI Training sul tuo ambiente di lavoro.",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_pl_PL.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_title": "Zainstaluj klienta AI Training",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_pl_PL.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_description": "Dowiedz się, jak szybko zainstalować klienta AI Training w Twoim środowisku pracy.",
packages/manager/modules/pci/src/projects/project/training/onboarding/translations/Messages_pt_PT.json: "pci_projects_project_training_onboarding_guides_install-eai-cli_title": "Instalar o cliente AI Training",
```

All `install-eai-cli` must be replaced by `install-cli`.